### PR TITLE
[RestExchange] add get_leverage_tiers

### PR DESCRIPTION
**Pull request by techfreaque**

### CODE REVIEW

File: octobot_trading/exchanges/adapters/abstract_adapter.py [hunk 1]
```diff
@@ -211,6 +216,13 @@ def fix_funding_rate_history(self, raw, **kwargs):
     def parse_funding_rate_history(self, fixed, **kwargs):
         raise NotImplementedError("parse_funding_rate_history is not implemented")
 
+    def fix_leverage_tiers(self, raw, **kwargs):
+        # add generic logic if necessary
+        return raw
+
+    def parse_leverage_tiers(self, fixed, **kwargs):
+        raise NotImplementedError("parse_funding_rate_history is not implemented")
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/adapters/abstract_adapter.py:224]:
```suggestion
        raise NotImplementedError(parse_leverage_tiers is not implemented")
```

**Review by GuillaumeDSM — COMMENTED** [review]
👍 
Should we add enums to store the leverage tiers info within the bot ? I mean at least as a uniform dict

**Comment by techfreaque** [comment]
> 👍 Should we add enums to store the leverage tiers info within the bot ? I mean at least as a uniform dict

I added enums and an adapter for it

File: octobot_trading/enums.py [hunk 2]
```diff
@@ -366,6 +366,16 @@ class ExchangeConstantsLeveragePropertyColumns(enum.Enum):
     RAW = "raw"
 
 
+class ExchangeConstantsLeverageTiersColumns(enum.Enum):
+    TIER = "tier"
+    CURRENCY = "currency"
+    MIN_NOTIONAL = "minNotional"
```
Comment (GuillaumeDSM) [inline octobot_trading/enums.py:372]: can we use snake_case for octobot unified enums plz ? try to follow this principle

**Review by GuillaumeDSM — COMMENTED** [review]
good for me once on snake_case for unified enums and CI passes

File: octobot_trading/enums.py (hunk 2 above)
Comment (techfreaque) [reply octobot_trading/enums.py]: done

**Review by GuillaumeDSM — APPROVED** [review]

File: octobot_trading/enums.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot_trading/enums.py]: thanks !

**Comment by GuillaumeDSM** [comment]
Good for me, @Herklos I let you merge this when you approve !

File: octobot_trading/exchanges/types/rest_exchange.py [hunk 3]
```diff
@@ -637,6 +637,13 @@ async def get_symbol_leverage(self, symbol: str, **kwargs: dict):
         """
         raise NotImplementedError("get_symbol_leverage is not implemented")
 
+    async def get_leverage_tiers(self, symbols: str = None, **kwargs: dict):
```
Comment (Herklos) [inline octobot_trading/exchanges/types/rest_exchange.py:640]: Can you add the method declaration in AbstractExchange please?

Comment (techfreaque) [reply octobot_trading/exchanges/types/rest_exchange.py:640]: done

**Review by Herklos — APPROVED** [review]

**Pull request merged**